.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.7]
```

Comment:
What would be required to also support 3.6 and 3.8?

Reply:
As far as I can tell, Python 3.8 just works. However, Python 3.6 is not supported by `tifffile` anymore.

Reply:
Then, we should check with the downstream projects (mostly voxelytics) that 3.6 will not be used anymore.

Reply:
And add 3.8 to the CI for testing.